importer/importer.py

```diff
@@ -3,21 +3,21 @@
 
 class DataImporter:
 
-    BASE_DATA_PATH = "/lustre/scratch118/infgen/pathogen/pathpipe/external_seq_data"
+    BASE_DATA_PATH = "/lustre/scratch118/infgen/pathogen/pathpipe/external_seq_data" # Only used in tests
 
     @staticmethod
     def new_instance(base: str, ticket: int, database: str):
         loader_list = []
         destination = "%s/%d" % (base, ticket)
-        for index, file in enumerate(glob.glob(f"{DataImporter.BASE_DATA_PATH}/{ticket}/external_{ticket}_*.xlsx",
+        for index, file in enumerate(glob.glob(f"{base}/{ticket}/external_{ticket}_*.xls",
                                                recursive=False)):
             command = DataImporter(destination, ticket, index, database)
             loader_list.append(command)
         return loader_list
     
     @staticmethod
-    def get_complete_manifest_for_ticket(ticket: int):
-        return f"{DataImporter.BASE_DATA_PATH}/{ticket}/complete_external_{ticket}.xlsx"
+    def get_complete_manifest_for_ticket(ticket: int, base: str):
+        return f"{base}/{ticket}/complete_external_{ticket}.xlsx"
```

Comment:
Do you know of any reason why we use xlsx format for the complete manifest? I don't think it matters, but I was just curious. I see that this was the behaviour or 1.0.14. I can see the logic in supporting xls or xlsx user input. However, I don't know why we don't stick with xls output throughout. I suppose it could be that if a user supplies xlsx input, then there may be formatting changes if we convert back to xls and these changes potentially interfere with downstream processing? It adds a bit of complexity having to deal with 2 formats for intermediate files, but the fix itself looks fine to me.

Reply:
The reason is that reading it back in only seems to work for xlsx but not xls. The whole underlying change that broke it in the first place was adding xlsx capability, so I guess everything got switched to xlsx.